migration/migrator/migrations/course/20200608034200_peer_feedback.py

```diff
@@ -0,0 +1,105 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+
+    # Create overall comment table
+    database.execute(
+        """
+       CREATE TABLE IF NOT EXISTS peer_feedback (
+            uid integer NOT NULL,
+            grader_id character varying(255) NOT NULL,
+            user_id character varying(255),
+            team_id character varying(255),
+            g_id character varying(255) NOT NULL,
+            feedback_full character varying(255),
```

Comment:
Why is this `feedback_full`? Is there a `feedback_short`?

Reply:
feedback_id is what the sytem uses internally to id all the different feedbacks. feedback_full is what users see as feedback. This allows for ids to maintain standards regardless of the actual feedback